app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1> Project Runway Contestants</h1>
+
+
+<% @contestants.each do |contestant|%>
+  <section id="contestant-<%= contestant.id %>">
+    <h3><%= contestant.name %></h3>
+  <p>Projects:</p>
+    <% contestant.projects.each do |project|%>
```

Comment:
This works and nice job. If you did not want to do nested iterating here you could build a method in your `contestant` model and then just call that method on your contestant from the block variable on line 4.